alwaysontop/render.js

```diff
@@ -0,0 +1,138 @@
+/* global __dirname */
+const { remote } = require('electron');
+
+const path = require("path");
+const url = require("url");
+
+/**
+ * URL for index.html which will be our entry point.
+ */
+const alwaysOnTopURL = url.format({
+    pathname: path.join(__dirname, "alwaysontop.html"),
+    protocol: "file:",
+    slashes: true
+});
+
+/**
+ * Implements the always on top functionality for the render process.
+ */
+class AlwaysOnTop {
+    /**
+     * Creates new instance.
+     *
+     * @param {JitsiIFrameApi} api - the Jitsi Meet iframe api object.
+     */
+    constructor(api) {
+        this._updateLargeVideoSrc = this._updateLargeVideoSrc.bind(this);
+        this._openAlwaysOnTopWindow = this._openAlwaysOnTopWindow.bind(this);
+        this._closeAlwaysOnTopWindow = this._closeAlwaysOnTopWindow.bind(this);
+
+        this._api = api;
+        this._jitsiMeetElectronWindow
+            = remote.getCurrentWindow();
+
+        if (!api) {
+            throw new Error('Wrong arguments!');
+        }
+        this._jitsiMeetElectronWindow.on('blur', this._openAlwaysOnTopWindow);
+        this._jitsiMeetElectronWindow.on('focus', this._closeAlwaysOnTopWindow);
+        this._jitsiMeetElectronWindow.on('close', this._closeAlwaysOnTopWindow);
+    }
+
+    /**
+     * Getter for the large video element in Jitsi Meet.
+     *
+     * @returns {HTMLElement|undefined} the large video.
+     */
+    get _jitsiMeetLargeVideo() {
+        const iframe = this._api.getIFrame();
+        if(!iframe || !iframe.contentWindow || !iframe.contentWindow.document) {
+            return;
+        }
+        return iframe.contentWindow.document.getElementById('largeVideo');
```

Comment:
Good point! 

Although I can't think of anything else that will be very different. Any ideas?

The important part here is that even if the external resources for the toolbar are not loaded for some reason at least the video will be displayed.